### ISSUE TITLE
choose_action

### ISSUE THREAD

**Issue by hwz9612** [quantumiracle/MARS#2]
Hi,
Now I train my env by using nash_ppo. The picture below is my environment.
![image](https://github.com/quantumiracle/MARS/assets/35334392/acded09a-6d3a-4306-911f-559a6b108dc1)
And I find that the policy hardly changes in each iteration. Why?
Thanks.

**Comment by quantumiracle** [comment]
Thanks for your question.

First, Nash PPO is only an internally tested algorithm that is not officially published, and it's based on gradient descent ascent for two agents to hopefully converge to the Nash solution, but it's not guaranteed.

For your case, I would suggest to try a few things: 1. if PPO learns correct policy for your environment; 2. check the gradient flow on polices to see whether it is caused by small gradients, and why small gradients appear in your case.

**Comment by hwz9612** [comment]
Hi,
   Thanks for your suggestions. And What exactly do I need to do？

**Comment by hwz9612** [comment]
Hi,
   Just as you said, I need to modify parameter of nash_ppo.

**Comment by hwz9612** [comment]
Hi, 
  Just as you said, do I need to modify parameter of nash ppo? After two or three iterations, the actions of agent hardly change. Just as the following picture shows, the actions of agent is 19 and 17.
![image](https://github.com/quantumiracle/MARS/assets/35334392/c0dece4b-da3e-4e1d-baef-9bfd2223e663)

Why does this happen？

**Comment by quantumiracle** [comment]
Hi maybe send me an email and collaborate on the problem: zhding96@gmail.com
